verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -0,0 +1,169 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "indexing_facts_tree_extractor.h"
+#include "verilog/CST/declaration.h"
+#include "verilog/CST/module.h"
+#include "verilog/analysis/verilog_analyzer.h"
+
+namespace verilog {
+namespace kythe {
+
+void DebugSyntaxTree(const verible::SyntaxTreeLeaf& leaf) {
```

Comment:
This function is not exported (it's not present in header). Put it in anonymous namespace:

```
namespace {
  void Debug...
}
```

Same for the rest of non-exported functions.

Reply:
We also accept functions with `static` linkage in this codebase.